runtime/src/storable_accounts.rs

```diff
@@ -154,40 +151,6 @@ impl<'a> StorableAccounts<'a, StoredAccountMeta<'a>>
     fn include_slot_in_hash(&self) -> IncludeSlotInHash {
         self.2
     }
-}
-
-/// this tuple contains a single different source slot that applies to all accounts
-/// accounts are StoredAccountMeta
-impl<'a> StorableAccounts<'a, StoredAccountMeta<'a>>
-    for (
-        Slot,
-        &'a [&'a StoredAccountMeta<'a>],
-        IncludeSlotInHash,
-        Slot,
-    )
-{
```

Comment:
the diff is confusing. The one a few lines down here:
```
&'a [&'a FoundStoredAccount<'a>],
```
became 
```
&'a [&'a StoredAccountMeta<'a>],
```
because I impl'd the function `has_hash_and_write_version` in the `&'a [&'a StoredAccountMeta<'a>],` version.

I can peel that impl off separately if you want and then the diff will be more clear what happened.

Reply:
When I was looking it over before sending it to you I saw that and asked the same question... I almost commented this in the pr before I sent it to you.